src/blocks/file-blocks/code/index.tsx

```diff
@@ -79,15 +79,15 @@ export default function (props: FileBlockProps) {
   const {
     content,
     context: { path },
-    onRequestUpdateContent,
+    isEditable,
+    onUpdateContent,
   } = props;
 
   const editorRef = React.useRef<HTMLDivElement>(null);
 
-  // track `onRequestUpdateContent` so the `updateListener` callback gets the latest one.
-  const onRequestUpdateContentRef =
-    React.useRef<typeof onRequestUpdateContent>();
-  onRequestUpdateContentRef.current = onRequestUpdateContent;
+  // track `onUpdateContent` so the `updateListener` callback gets the latest one.
+  const onUpdateContentRef = React.useRef<typeof onUpdateContent>();
+  onUpdateContentRef.current = onUpdateContent;
```

Comment:
do we still need to do this after the changes in /blocks?

Reply:
no, will pull it!